### PR TITLE
A better fix for the GUI update loop #185 

### PR DESCRIPTION
**Pull request by mariobuikhuizen**
A better fix for #185.

Reverts 95f3e8e and replaces v-treeview with a list of checkboxes.

### CODE REVIEW

**Review by javerbukh — APPROVED** [review]
Used this on cubeviz, mosviz, and specviz and it works great! Clicked repeatedly on checkboxes trying to trigger the bug and even though it occasionally takes a little bit to catch up, the application does not get stuck in a loop.

File: jdaviz/container.vue [hunk 1]
```diff
@@ -75,16 +75,16 @@
...
-                  ></v-treeview>
+                  <v-checkbox
+                    v-for="item in dataItems" :key="item.id" :label="item.name" dense hide-details
+                    :input-value="viewer.selected_data_items.includes(item.id)"
+                    @change="$emit('data-item-selected', {
+                      id: viewer.id,
+                      item_id: item.id,
+                      checked: $event
+                    })"
+                    class="pl-2"
```
Comment (rosteen) [inline jdaviz/container.vue:86]:
```suggestion
                    class="pl-4"
```
Tiny change, I think this looks better with a little more padding

**Review by rosteen — CHANGES_REQUESTED** [review]
I requested one very minor UI change, other than that this looks great. Tested programmatically changing selected data and it works, neither of the previous bugs occur.

File: jdaviz/container.vue (hunk 1 above)
Comment (mariobuikhuizen) [reply jdaviz/container.vue]: 👍

**Review by rosteen — APPROVED** [review]

**Comment by rosteen** [comment]
Merging, thanks Mario!

**Pull request merged**